crates/ark/src/variables/variable.rs

```diff
@@ -312,6 +320,34 @@ impl WorkspaceVariableDisplayValue {
         log::trace!("Error while formatting variable: {err:?}");
         Self::new(String::from("??"), true)
     }
+
+    fn from_untruncated_string(mut value: String) -> Self {
+        let Some((index, _)) = value.char_indices().nth(MAX_DISPLAY_VALUE_LENGTH) else {
+            return Self::new(value, false);
+        };
+
+        // If an index is found, truncate the string to that index
+        value.truncate(index);
+        Self::new(value, true)
+    }
+
+    fn try_from_method(value: SEXP) -> Option<Self> {
+        let display_value =
+            ArkGenerics::VariableDisplayValue.try_dispatch::<String>(value, vec![RArgument::new(
+                "width",
+                RObject::from(MAX_DISPLAY_VALUE_LENGTH as i32),
+            )]);
+
+        let display_value = unwrap!(display_value, Err(err) => {
+            log::error!("Failed to apply 'ark_variable_display_value': {err:?}");
```

Comment:
```suggestion
            log::error!("Failed to apply 'positron_variable_display_value': {err:?}");
```

or `ark_positron_`, whatever we decide

Reply:
Fixed in https://github.com/posit-dev/ark/pull/560/commits/fdee0f42f4727615f5f3b31a4aa41c061f849781